icbc-adapter/src/Pssg.Dmf.IcbcAdapter/Startup.cs

```diff
@@ -228,7 +228,7 @@ public void ConfigureServices(IServiceCollection services)
 
                 var httpClient = new HttpClient(httpClientHandler) 
                 { 
-                    Timeout = TimeSpan.FromMinutes(30)
+                    Timeout = TimeSpan.FromMinutes(60)
```

Comment:
Do we need this to be much larger?   We should test with a large dataset

Reply:
Changed the timeout to 30 minutes